src/test/java/net/openhft/chronicle/wire/WireTests.java

```diff
@@ -119,14 +116,29 @@ public void testLenientTypeLiteral() {
 
     @Test
     public void testDate() {
+        Locale defLocale = Locale.getDefault();
+
+        Locale.setDefault(Locale.ITALY);
+
         final Bytes b = Bytes.elasticByteBuffer();
```

Comment:
should this be elasticHeap.... so we don't have to worry about releasing?

Reply:
I'm not sure but I will skip changing the whole test class code for now